push-image-ghcr/src/index.ts

```diff
@@ -0,0 +1,72 @@
+import { getInput, info, setFailed, warning } from '@actions/core'
+import { exec } from '@actions/exec'
+import { context} from '@actions/github'
+import {
+  getRepository,
+  getToken,
+} from '../../lib/github'
+import {
+  setComponentVersion,
+  setLabels,
+  setTags,
+} from './metadata'
+
+const METADATA_FILENAME = 'metadata.json'
+
+async function run() {
+  const { component: defaultComponent } = getRepository()
```

Comment:
please align with inputs in README

Reply:
I not sure what to align here

Reply:
My bad, should have commented on the next line

```
const component = getInput('component') || defaultComponent
```

The `"component"` input is not documented